### DISCUSSION TITLE
What k8s versions are supported?

### DISCUSSION THREAD

**Discussion by motiumansky** [aquasecurity/kube-bench#908] [Questions and Help] [answered]
We are considering to run kube-bench on our EKS. It is unclear to us what versions are supported.

Thanks,
Moti

**Comment by yoavrotems** [comment, marked as answer]
Hey @motiumansky you can see platforms and supported versions in the documentation files and site, 
https://aquasecurity.github.io/kube-bench/dev/platforms/

**Reply by motiumansky** [reply]
Thank you for your quick reply!

Does this mean that latest supported k8s version is 1.16?

Moti

**Reply by yoavrotems** [reply]
Yes the latest CIS we support is 1.6 which relevant to kubernetes 1.16-1.18
In the next few days we are going to add a support for the new CIS benchmark which is relevant to 1.19-1.20. 
There is an issue in the subject #897 😄

**Reply by motiumansky** [reply]
Thank you for your update!

I see that you have merged the "CIS Kubernetes V1.20 Benchmark v1.0.0" PR.

Guess that https://aquasecurity.github.io/kube-bench/dev/platforms/ wasn't updated yet?

Moti

**Reply by yoavrotems** [reply]
Yes we merged it and we will release it officially with new kube-bench version same as we did for the last version https://aquasecurity.github.io/kube-bench/v0.6.3/platforms/ v0.7.0 will have its own documentation version 😸

**Reply by motiumansky** [reply]
Thank you for your update.

Do you know when v0.7.0 will be available?

Moti

**Reply by yoavrotems** [reply]
Its still a working process, but a rough ETA I would say in a month or so, I could update when we will have more accurate date

**Reply by motiumansky** [reply]
Hi Yoav

Thank you for your update!

Maybe you have the ETA for v0.7.0?

Moti

**Comment by motiumansky** [comment]
Hi Yoav

I see that v0.65 has the 1.20 support.

Could you please share the URL of the relevant platform  page? (like https://aquasecurity.github.io/kube-bench/v0.6.3/platforms/)

Thanks,
Moti

**Reply by yoavrotems** [reply]
Hey thanks for informing us on the missing docs, you can now access 
https://aquasecurity.github.io/kube-bench/v0.6.5/platforms/ 

Thanks, 
Yoav
